docs-ref-services/service-map.md

```diff
@@ -16,17 +16,25 @@ ms.service: Service Map
 # Azure Service Map packages for Node.js
 
 ## Overview
+
 Service Map automatically discovers application components on Windows and Linux systems and maps the communication between services. Service Map shows connections between servers, processes, and ports across any TCP-connected architecture, with no configuration required other than the installation of an agent.
 
 Learn more about [Azure Service Map](https://docs.microsoft.com/en-us/azure/operations-management-suite/operations-management-suite-service-map).
 
-## Management Package
-### Install the npm module
+## Management package
+
+### Install npm module
+
+Install the Azure Service map npm module

```

Comment:
Should this be `Azure Service Map npm package`? Capitalize the full product name and replace "module" with "package"?